### ISSUE TITLE
Layout elements should be aria-hidden

### ISSUE THREAD

**Issue by jackahl** [kitconcept/volto-light-theme#411]
layout elements such as "|" in header or ">" in Breadcrumbs should not be read out by screen-readers. Therefore we should add aria-hidden attributes to them, where necessary
